h_periodic/lms_beat.py

```diff
@@ -32,6 +32,10 @@
             "task": "lms.tasks.hubspot.refresh_hubspot_data",
             "schedule": crontab(hour=13),
         },
+        "export_companies_contract_billables": {
+            "task": "lms.tasks.hubspot.export_companies_contract_billables",
+            "schedule": crontab(hour=14),
```

Comment:
Once a day, same as refresh_hubspot_data but an hour later.

There are many moving parts here to determine what/when will be synced (changes in HubSpot, time of the month...)

We just call these once a day.